.scripts/build_steps.sh

```diff
@@ -6,8 +6,14 @@
 # benefit from the improvement.
 
 set -xeuo pipefail
-export PYTHONUNBUFFERED=1
 export FEEDSTOCK_ROOT="${FEEDSTOCK_ROOT:-/home/conda/feedstock_root}"
+source ${FEEDSTOCK_ROOT}/.scripts/logging_utils.sh
+
+
+endgroup "Start Docker"
```

Comment:
why is the first think that is logged a `endgroup`, should this by any chance be a `startgroup`? (I don't know a lot about the output that is needed in the logging of the different tools)